File: categorias/iniciante/python/1040.py
# -*- coding: utf-8 -*-

n = input()
n0 = n.split()
n1, n2, n3, n4 = n0
m = ((n1*2)+(n2*3)+(n3*4)+(n4*1))/10

print('Media: {:.1f}'.format(m))

if (m >= 7.0):
    print('Aluno aprovado.')
elif (m < 5.0):
    print('Aluno reprovado.')
elif (5.0 <= m <= 6.9):
    print('Aluno em exame.')
    print('Nota do exame: {:.1f}'.format(e))
    e = float(input())
    mf = (e+m)/2
    if (mf >= 5.0):
        print('Aluno aprovado.')
    elif (mf <= 4.9):
        print('Aluno reprovado.')
    print('Media final: {:.1f}'.format(mf))

File: categorias/iniciante/c++/1040.cpp
#include <iostream>

using namespace std;

int main() {
    double n1, n2, n3, n4, media, r;
    cout.precision(1);
    cout.setf(ios::fixed);

    cin >> n1 >> n2 >> n3 >> n4;
    media = (n1*2 + n2*3 + n3*4 + n4)/10;
    cout << "Media: " << media << "\n";
    if (media >= 7) {
        cout << "Aluno aprovado.\n";
    } else if (media < 5) {
        cout << "Aluno reprovado.\n";
    } else {
        cout << "Aluno em exame.\n";
        cin >> r;
        cout << "Nota do exame: " << r << "\n";
        if (r >= 5) {
            cout << "Aluno aprovado.\n";
        } else {
            cout << "Aluno reprovado.\n";
        }
        cout << "Media final: " << (r+media)/2 << "\n";
    }

    return 0;
}